.editorconfig

```diff
@@ -0,0 +1,30 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
+tab_width = 4
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.go]
+indent_style = tab
```

Comment:
Tab? :thinking: I guess this has to match go fmt.

Reply:
Exactly, go uses tabs ^^

Reply:
Srsly? :exploding_head:

Reply:
Yes ^^

> Gofmt formats Go programs. It uses tabs for indentation and blanks for alignment

https://golang.org/cmd/gofmt/

Reply:
I guess this shows one thing clearly: I like that `go fmt` takes care of the formatting so that I don't need to know/care whether its tabs or spaces :smile:

Reply:
Yes!

Having the language define how code should be formatted is a big plus if you ask me :laughing: 
No more discussion wasted on tabs/spaces :laughing: